vendor/engines/split_accounts/spec/engine_helper.rb

```diff
@@ -0,0 +1,9 @@
+# Rspec configurations for this engine
+RSpec.configure do |config|
+  
+  # Skip split_accounts tests if split_accounts feature is disabled
+  unless SettingsHelper.feature_on?(:split_accounts)
```

Comment:
We've tended to use the `feature_setting` metadata to turn on features as they're needed. I don't have a problem not running tests in one of the downstream branches, but I don't like skipping tests in open.